src/code/z_scene_table.c

```diff
@@ -215,9 +215,9 @@ void Scene_DrawConfigDefault(PlayState* play) {
     CLOSE_DISPS(play->state.gfxCtx, "../z_scene_table.c", 4735);
 }
 
-void* D_8012A2F8[] = {
-    gYdanTex_00BA18,
-    gYdanTex_00CA18,
+void* sDekuTreeEntranceTextures[] = {
+    gDekuTreeDayEntranceTex,
+    gDekuTreeNightEntranceTex,
 };
 
 void Scene_DrawConfigDekuTree(PlayState* play) {
```

Comment:
For very localized data, like all these texture arrays, I would personally not be against inlining them into the function. as in
```suggestion
void Scene_DrawConfigDekuTree(PlayState* play) {
    static void* sDekuTreeEntranceTextures[] = {
        gDekuTreeDayEntranceTex,
        gDekuTreeNightEntranceTex,
    };
    ...
```
It has the downside of not appearing in the map file, but I like the organizational benefit and dont think something like this would need to be looked up too often by anybody